templates/items/details/details-facility.hbs

```diff
@@ -61,3 +54,66 @@
         </div>
     </div>
 </fieldset>
+{{/if}}
+
+{{#if (and (eq source.type.value "special") (eq source.order "craft"))}}
+<fieldset>
+    <legend>{{ localize "DND5E.FACILITY.Orders.craft.present" }}</legend>
+    <ul class="separated-list facility-craft">
```

Comment:
It would be good to have a hint here to indicate that this is the item currently being crafted, not a list of items that can be crafted.